src/utils/atomicSwap.js

```diff
@@ -140,14 +140,14 @@ export function calculateExhibitionData(partialTx, cachedTokens, wallet) {
                 tokenUid,
                 symbol: '',
                 name: '',
-                status: PROPOSAL_DOWNLOAD_STATUS.LOADING
+                status: PROPOSAL_DOWNLOAD_STATUS.LOADING,
             };
         }
 
         return cachedTokens[tokenUid];
     }
 
-    const txProposal = new PartialTxProposal.fromPartialTx(partialTx.serialize(), wallet.getNetworkObject());
```

Comment:
This is a bugfix: `fromPartialTx` is not a constructor and should not be called with `new`.